.github/workflows/linkcheck.yml

```diff
@@ -0,0 +1,26 @@
+name: Linkcheck
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  style:
+    name: Check external links
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install .[doc]
+          sudo apt-get -y install pandoc
+      - name: Check external links
+        working-directory: doc
+        run: make linkcheck
```

Comment:
Seems like it just some backdoor to sidestep the linkcheck. In my opinion it just puts unnecessary load on the github actions.

Reply:
No, I moved the linkcheck job from the CI workflow to this separate workflow.

The thing is, if only one of the jobs in a workflow fails, the whole workflow has to be restarted. So actually this decreases the pressure on Actions, because we regularly restart the linkcheck.